app/models/social-link.js

```diff
@@ -4,14 +4,15 @@ import attr from 'ember-data/attr';
 import ModelBase from 'open-event-frontend/models/base';
 import { belongsTo } from 'ember-data/relationships';
 import { computedSegmentedLink, socialPlatforms } from 'open-event-frontend/utils/computed-helpers';
+import { capitalize } from 'lodash-es';
 
 export default ModelBase.extend({
   name       : attr('string'),
   link       : attr('string'),
   identifier : attr('string'), // used for providing css id for URL validations.
 
   event: belongsTo('event'),
-
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6232875)

Reply:
Please don't resolve the conversations. They are resolved automatically